hw/top_earlgrey/lint/top_earlgrey_dv_lint_cfgs.hjson

```diff
@@ -0,0 +1,80 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{
+
+  // This is the primary cfg hjson for DV linting. It imports ALL individual lint
+  // cfgs of the IPs DV environments and the full chip DV environment for top_earlgrey.
+  // This enables to run them all as a regression in one shot.
+  name: top_earlgrey_dv_batch
+
+  import_cfgs:      [// common server configuration for results upload
+                     "{proj_root}/hw/data/common_project_cfg.hjson"
+                     // tool-specific configuration
+                     "{proj_root}/hw/lint/data/{tool}.hjson"]
+
+  // Different dashboard output path for each tool
+  rel_path: "hw/top_earlgrey/dv_lint/{tool}"
+
+  use_cfgs: [{    name: aes_sim
+                  fusesoc_core: lowrisc:dv:aes_sim
+                  import_cfgs: ["{proj_root}/hw/lint/data/common_lint_cfg.hjson"]
+                  rel_path: "hw/dv/aes_sim/dv_lint/{tool}"
+             },
+             {    name: alert_handler_sim
+                  fusesoc_core: lowrisc:dv:alert_handler_generic_sim
```

Comment:
We want to be running on `lowrisc:dv:alert_handler_sim` instead here (as the cfg list hjson file name suggests), which is located in `top_earlgrey/ip/alert_handler`. Likewise for all templated IPs that take a specific parameter set for `top_earlgrey`.

Reply:
makes sense - changed. It looks like all other templated IPs are currently verified using FPV only, hence this is the only one that needs to be amended.